src/beman/optional26/tests/optional.t.cpp

```diff
@@ -868,3 +868,23 @@ TEST(OptionalTest, HashTest) {
         EXPECT_EQ(h1, h2);
     }
 }
+
+TEST(OptionalTest, CanHoldValueOfImmovableType) {
+    struct immovable {
```

Comment:
Please move this test class in https://github.com/bemanproject/optional26/blob/main/src/beman/optional26/tests/test_types.hpp

Reply:
Done!